lms/templates/logout.html

```diff
@@ -0,0 +1,23 @@
+{% extends "main_django.html" %}
```

Comment:
Heh, I think you're preaching to the choir here -- notice I didn't really say to do anything about that...